tutorials/builtin_algorithms/GCP_AI_Platform_Image_Classification_Built_in_Algorithm.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>It would be ideal to remove the dependency on tf.contrib; if not, the tutorial should specify the TensorFlow version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ml-on-gcp/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>